PLAN.md

```
@@ -1,259 +1,64 @@
+| Vite | 6.4.x |
+| TypeScript | 5.9.x |
+| Tailwind CSS | 4.3.x (via `@tailwindcss/vite`, CSS-first) |
+| Vitest | 4.0.x |
+| vue-tsc | 3.2.x |
+
+- `npm run dev` — dev server
+- `npm run build` — `vue-tsc --noEmit && vite build`
+- `npm test` — Vitest (44 tests)
+
+## Architecture
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)